### PR TITLE
Register to an event for using sendUserProperties on time

### PR DESCRIPTION
**Pull request by meravzanany**
## Description
Use the LocalBroadcastManager to register for an event that will be sent from the login plugin and upon receiving, we will know we can call the sendUserProperties method

## Checklist

* [ ] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [X] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

### CODE REVIEW

File: Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java [hunk 1]
```diff
@@ -40,6 +45,8 @@
     public static final String GA_PREFIX = "ga_";
     public static final String SEND_USER_DATA = "Send_User_Data";
     public static final String USER_ID = "user_id";
+    public static final String SEND_BROADCAST_ACTION = "send_broadcast_from_rn";
```
Comment (snooky23) [inline Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java:48]: Please give it a more meaningful name.
What was the old SDK naming (missing interface method)?
Maybe we can use that :)

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java (hunk 1 above)
Comment (snooky23) [reply Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java:48]: Please be aware that in the following p.r: https://github.com/applicaster/zapp-login-plugin-emmys-cognito/pull/59/files
the key name is `userProperties`.

Comment (meravz82) [reply Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java:48]: You're confusing the action and the key...
The intent action is send_broadcast_from_rn (I'm registered to this action) and upon receiving an event, we have the key and properties that can be anything we want.
btw - 
val SEND_BROADCAST_ACTION = "send_broadcast_from_rn" is taken from here:
https://github.com/applicaster-plugins/react-native-broadcast-manager/pull/1/files - RNBroadcasterManager.kt

Comment (meravz82) [reply Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java:48]: Made another change - now I only call sendUserProperties when the EVENT_NAME  is "userProperties"

**Review by snooky23 — APPROVED** [review]

**Review by lomudi — APPROVED** [review]

**Pull request merged**